libecs/tests/Core/Scene/test_load.cxx

```diff
@@ -0,0 +1,59 @@
+#include "pivot/ecs/Core/Scene.hxx"
+#include <catch2/catch_test_macros.hpp>
+#include <nlohmann/json.hpp>
+#include <pivot/ecs/Components/Gravity.hxx>
+#include <pivot/ecs/Components/RigidBody.hxx>
+#include <pivot/ecs/Components/Tag.hxx>
+
+using namespace nlohmann;
+using namespace pivot::ecs::data;
+
+void test_load_scene_system(const pivot::ecs::systems::Description &, pivot::ecs::component::ArrayCombination &,
+                            const pivot::ecs::event::EventWithComponent &)
+{
+}
+
+TEST_CASE("Load the scene", "[Scene][Load]")
+{
+    json obj = json::parse("{\"components\": [{\"Gravity\": {\"force\": [0.0,0.0,0.0]},\"RigidBody\": "
+                           "{\"acceleration\": [0.0,0.0,0.0],\"velocity\": [0.0,0.0,0.0]},\"Tag\": {\"name\": "
+                           "\"yolo\"}}],\"name\": \"Default\",\"systems\":[\"Test Description\"]}");
```

Comment:
You can use a raw string https://stackoverflow.com/questions/56710024/what-is-a-raw-string
```suggestion
    json obj = json::parse(R"({"components": [{"Gravity": {"force": [0.0,0.0,0.0]},"RigidBody":
                           {"acceleration": [0.0,0.0,0.0],"velocity": [0.0,0.0,0.0]},"Tag": {"name":
                           "yolo"}}],"name": "Default","systems":["Test Description"]})");
```